internal/xds/util.go

```diff
@@ -16,9 +16,12 @@ package xds
 import (
 	"strings"
 
+	xds "github.com/envoyproxy/go-control-plane/pkg/cache/types"
 	"k8s.io/apimachinery/pkg/types"
 )
 
+type ResourceType = xds.ResponseType
```

Comment:
Nit: now that the resource types are encapsulated by the snapshot handler, I don't think you need this?